Core/Core/Extensions/NSErrorExtensions.swift

```diff
@@ -22,6 +22,11 @@ extension NSError {
     public struct Constants {
         static let domain = "com.instructure"
         static let internalError = "Internal Error"
+
+        static let unauthorized = 1000
+        static let forbidden = 1001
+        static let notFound = 1002
+        static let unexpected = 2000
```

Comment:
I created a new struct called `HttpError` and placed it above `APIError` for visibility. Once we do an overhaul of error handling it might be easier to refactor this way.